Validation/RecoVertex/src/PrimaryVertexAnalyzer4PUSlimmed.cc

```diff
@@ -22,6 +22,20 @@
 
 #include <numeric>
 
+namespace {
+  template <typename T, size_t N>
+  std::array<T, N+1> makeLogBins(const double min, const double max) {
+    const double minLog10 = std::log10(min);
+    const double maxLog10 = std::log10(max);
+    const double width = (maxLog10-minLog10)/N;
+    std::array<T, N+1> ret;
```

Comment:
Thanks, I'll take a note (even though the function is called only once from `bookHistograms()`).